script/openqa-bootstrap

```diff
@@ -101,12 +103,12 @@ retry -e -s 30 -r 7 -- sh -c "zypper -n --gpg-auto-import-keys ref && zypper -n
 # setup database
 chown -R postgres: /var/lib/pgsql/ # fix broken postgres working dir permissions in the nspawn container
 start-database
-su postgres -c "/usr/share/openqa/script/setup-db" "$dbuser" "$dbname"
+su postgres -c "$OPENQA_DIR/script/setup-db" "$dbuser" "$dbname"
 
 # setup webserver and fake-auth
 proxy_args=""
 [[ -n "$setup_web_proxy" ]] && proxy_args="--proxy=$setup_web_proxy"
-setup=/usr/share/openqa/script/configure-web-proxy
+setup="$OPENQA_DIR/script/configure-web-proxy"
```

Comment:
You didn't have to use quotes on *that* use of the variable. (Expansion is not happening on assignments. so quoting is not needed here.)

You don't have to change this single line now, I'm just mentioning it for a better understanding.